src/styles/styles.less

```diff
@@ -2,38 +2,30 @@
 
 :root {
   // VARIABLES: DIMENSIONS FOR SPACERS, BORDERS, ETC
-  --widget-stroke-width: 1;
-
-  --clock-fill-opacity-hover: 0.6;
-
+  --ironsworn-widget-stroke-width: 1;
+  --ironsworn-clock-fill-opacity-hover: 0.6;
   --ironsworn-border-radius-sm: 2px;
   --ironsworn-border-radius-md: 3px;
   --ironsworn-border-radius-lg: 5px;
-
```

Comment:
Oh, losing these is a bit sad, it made this wall of rules more readable. 😞

Reply:
hm, good point - i'll tweak the rules to make them friendlier to arbitrary line breaks

Reply:
alternatively -- could add some comments, which would make the content more explicit and force an extra blank line 🤔